src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -323,15 +325,15 @@ public Builder addStdErrLineListener(ProcessOutputLineListener stdErrLineListene
      * The client listener of the process exit with code.
      */
     public Builder exitListener(ProcessExitListener exitListener) {
```

Comment:
yeah I had the clear in there, but it looked dumb. Let me put it back in, we can change the surface later.